ImporterFieldProcessorFile.php

```diff
@@ -157,9 +157,15 @@ public function processFieldFileCreateItem($uuid) {
 
     $clean_path = implode("/", $urlparts);
 
-    $url = $pend['scheme'] . "://" . $pend['host'] . $base_path . "/sites/default/files/" . $clean_path;
-
-    system_retrieve_file($url, $file->uri, 0, FILE_EXISTS_REPLACE);
+    // Try to fetch from the ACE environment first.
+    $url = $pend['scheme'] . "://" . $pend['host'] . $base_path . "/sites/jsa-content/files/" . $clean_path;
+    $fetch_success = system_retrieve_file($url, $file->uri, 0, FILE_EXISTS_REPLACE);
+   
+    // If the file download fails, try the SITES 1.x file path.
+    if(!$fetch_success) {
+      $url = $pend['scheme'] . "://" . $pend['host'] . $base_path . "/sites/default/files/" . $clean_path;
```

Comment:
The file won't exist at `https://jsa-content.stanford.edu/sites/default/files/path/to/file/filename.jpg` though, will it? I'm not sure that this is going to work.

Reply:
That is the path where we have been getting them for the last few years. We can slack chat about this tomorrow.

Reply:
Oh, I think I'm picking up what you're putting down. The former is for when DNS is pointing at ACE; the latter for when DNS is pointing at `sites-lb`.

Reply:
Yeah, instead of exposing the file_public_path so that the importer can fetch that information before fetching a file I just hit the two known assumptions one after the first one fails.